src/novaprobe.py

```diff
@@ -266,12 +266,18 @@ class ArgHolder(object):
                                 timeout=argholder.timeout)
         response.raise_for_status()
         for network in response.json()['networks']:
-            # assume first available and active network is ok
-            if network['status'] == 'ACTIVE':
+            # assume first available active network owned by the tenant is ok
+            if network['status'] == 'ACTIVE' and network['tenant_id'] == tenant_id:
```

Comment:
It should be better to use project_id. tenant_id will probably removed soon...

Reply:
The `tenant_id` is actually `project_id` when using Keystone V3, and the field returned by neutron is still called `tenant_id`, see: https://developer.openstack.org/api-ref/network/v2/?expanded=list-networks-detail#networks